h2o-core/src/main/java/water/util/FrameUtils.java

```diff
@@ -180,10 +180,24 @@ public Vecs2ArryTsk(int dim1, int dim2)
   }
 
   public static double [] asDoubles(Vec v){
-    if(v.length() > 100000) throw new IllegalArgumentException("Vec is too big to be extracted into array");
+    if(v.length() > 100_000) throw new IllegalArgumentException("Vec is too big to be extracted into array");
```

Comment:
Maybe the number 100 000 could be some constant defined globally?

Reply:
I created FrameUtils.MAX_VEC_NUM_ROWS_FOR_ARRAY_EXPORT constant for this purpose